README.md

```diff
@@ -237,6 +241,14 @@ The [response object from Gaxios](https://github.com/JustinBeckwith/gaxios/blob/
 
 The file's new metadata.
 
+#### .on('progress', function (progress) {})
+
+##### progress
+
+- Type: `Object`
+
+Progress event provides upload stats like Transferred Bytes and content length.
```

Comment:
Would you mind listing the property names, e.g. `progress.bytesWritten`?

Reply:
Added!!